packages/components/.eslintrc.js

```diff
@@ -39,40 +40,11 @@ module.exports = {
     ],
     '@stencil-community/strict-boolean-conditions': 'off',
     '@stencil-community/required-prefix': ['error', ['post-']],
-    '@stencil-community/async-methods': 'error',
-    '@stencil-community/decorators-context': 'error',
-    '@stencil-community/decorators-style': [
-      'error',
-      {
-        prop: 'inline',
-        state: 'inline',
-        element: 'inline',
-        event: 'inline',
-        method: 'multiline',
-        watch: 'multiline',
-        listen: 'multiline',
-      },
-    ],
     '@stencil-community/class-pattern': [
       'error',
       {
         pattern: '^Post.*(?!Component)$',
       },
     ],
-    '@stencil-community/element-type': 'error',
-    '@stencil-community/host-data-deprecated': 'error',
-    '@stencil-community/methods-must-be-public': 'error',
-    '@stencil-community/no-unused-watch': 'error',
-    '@stencil-community/own-methods-must-be-private': 'error',
-    '@stencil-community/own-props-must-be-private': 'error',
-    '@stencil-community/prefer-vdom-listener': 'error',
-    '@stencil-community/props-must-be-public': 'error',
-    '@stencil-community/props-must-be-readonly': 'error',
-    '@stencil-community/render-returns-host': 'error',
-    '@stencil-community/required-jsdoc': 'error',
-    '@stencil-community/reserved-member-names': 'error',
-    '@stencil-community/single-export': 'error',
-    '@stencil-community/strict-mutable': 'error',
-    'react/jsx-no-bind': 'off',
```

Comment:
Are you sure this is working? Because you did not have to ignore or disable the issue that you show me with form attachments methods (or I didn't see where), so it looks more that the rules are just disabled entirely.

Reply:
Ok good, no problem then.